### DISCUSSION TITLE
p/118495bafcc9414a989d5f8b72e9d618/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#60] [Announcements]
# p/118495bafcc9414a989d5f8b72e9d618/

漫长的暑假终于结束，可以继续出门旅行了。这周先去了附近的海陵岛，虽然一直素有坑人的水鱼岛之称，不过吾辈还是来玩了三天。  路线: 大角湾 => 大角湾夜滩 => 北洛秘境（沙滩）=> 马尾岛 => 滨海栈道 => 观海楼 => 十里银滩 大角湾刚到这里住在了大角湾，据说是岛上最方便的地方。来的时候已经是下午，便在附近走了走，把一块海滩围起来收费并且规定只允许在那

https://blog.rxliuli.com/p/118495bafcc9414a989d5f8b72e9d618/

**Comment by miaopass-future** [comment]
老哥在一个人旅行嘛，感觉你已经很久没有经受工作的磨练了

**Reply by rxliuli** [reply]
嗯，有段时间没工作了。话虽如此，短期仍然不考虑工作。
